src/python/pants/backend/terraform/tool.py

```diff
@@ -30,5 +42,37 @@ def default_known_versions(cls):
         ]
 
 
+@dataclass(frozen=True)
+class TerraformProcess:
+    """A request to invoke Terraform."""
+
+    args: tuple[str, ...]
+    description: str
+    input_digest: Digest = EMPTY_DIGEST
+    output_files: tuple[str, ...] = ()
+
+
+@rule(level=LogLevel.DEBUG)
```

Comment:
Imo not worth setting to Debug. The process itself will already have a Level associated with it, and I think `DownloadedExternalTool` has a non-trace workunit

Reply:
👍  Removed.